docs/installation/application.rst

```diff
@@ -0,0 +1,196 @@
+
+* **conda/channels**: List of conda channels that need to be searched for dependency installation. Only specify any conda channels that are apart from the default. 
+
+* **conda/dependencies**: List of python dependencies that need to be installed by conda. These may be entered in the format ``pyshp=2.0.0`` to download a specific version.
+
+* **pip**: A list of python dependencies that need to be installed by pip.
+
+* **post**: A list of shell scripts that you would want to run after the installation is complete. This option can be used to initialize workspaces/databases etc. These shell scripts should be present in the same directory as setup.py
+
+.. tip::
+
+    Run ``tethys gen install`` to create a blank template of this file. By default the file will be saved to the current working directory.
+
+.. _tethys_services_yml:
+
+services.yml 
+------------
+
+This file will be created by the portal administrator who has created/has access to all the service in the portal. This file will only be run by default if there is no portal services config file present (see below). However you can force the use of this file over the portal config by specifying the `--force-services` tag on the install command.  
+
+.. code-block:: python
+
+	# Do not commit this file. This file is portal specific.
+	version: 1.0
+	skip: false
+	interactive: false
+
+	# Set service params in the following format :
+	# app_service_setting_name(from your app.py): <service_name or id from list of installed services>
+	persistent:
+  	 catalog_db: hydroexplorer-persistent
+  	 second_db: main-persistent
+	wps:
+  	 wps_main: testWPS
+  	dataset:
+  	spatial:
+
+**services.yml Options:**
+
+* **version**: Indicated the version of the :file:`services.yml` file. Current default : 1.0
+* **skip**: If enabled, it will skip the installation of services. This option is set to `False` by default. 
+* **interactive**: If enabled, it will start an interactive mode that will let you select from existing portal services. This option is set to `False` by default. 
+
+
+* **persistent** : List of persistent store settings in the app and the service to link to each.
+* **dataset** : List of dataset settings in the app and the service to link to each.
+* **spatial** : List of spatial persistent store settings in the app and the service to link to each.
+* **wps** : List of Web Processing service settings in the app and the service to link to each. 
+
+Settings in each of the service sections above will need to be listed in the following format::
+
+	<app_service_setting_name> : <service_name or id>
+
+In the above example, ``catalog_db`` is the name of the service in your :file:`app.py` and ``hydroexplorer-persistent`` is the name of the service on the portal. 
+
+.. tip::
+
+    Run ``tethys gen services`` to create a blank template of this file. By default the file will be saved to the current working directory.
```

Comment:
> By default the file will be saved to the current working directory.
Does tethys expect to find this file is a specific location? If so, then I think the default should be that it writes to that location.

Reply:
Wait, this one is specific to apps, so there isn't a "default" location.